data/strings/strings.txt

```diff
@@ -23205,7 +23159,7 @@
 
   [never]
     comment = Generic «Never» string
-    tags = android,ios
+    tags = ios
```

Comment:
@kirylkaveryn is it hard to sync iOS texts with Android texts for this setting? https://github.com/organicmaps/organicmaps/pull/6125

Reply:
I will try to sync it!